live-examples/js-examples/number/number-min-value.js

```diff
@@ -1,12 +1,8 @@
-function multiply(x, y) {
-  if (x * y < Number.MIN_VALUE) {
-    return 'Process as -Infinity';
-  }
-  return (x * y);
-}
-
-console.log(multiply(5e-324, 1));
+console.log(Number.MIN_VALUE * .5);
 // expected output: 5e-324
 
-console.log(multiply(-1.7976931348623157e+308, 2));
-// expected output: Process as -Infinity
+console.log(Number.MIN_VALUE * (.5 * Number.EPSILON));
```

Comment:
This example you suggested is more  complex to understand than the current example. 
Also it uses another value (Number.EPSILON) to explain Number.MIN_VALUE, that I feel is not the best way to explain.

Reply:
The current example may be easy to understand, but it is wrong because it does not explain MIN_VALUE, it explains NEGATIVE_INFINITY which is not the subject of the example page.

But you are right. because (.5 * Number.EPSILON) is strange given the definition of Number.EPSILON.

How about...
console.log(Number.MIN_VALUE);
// expected output: 5e-324
console.log(Number.MIN_VALUE * 0.6);
// expected output: 5e-324
console.log(Number.MIN_VALUE * 0.5);
// expected output 0

Reply:
Sorry, my comment has a mistake.  I said (.5 * Number.EPSILON).  I missed the "+" in the original and saw a "*" instead.  So, though it correctly shows the smallest achievable granularity, maybe just using "0.6" would be sufficient without confusing things with EPSILON.

Reply:
Yes this look more clear. And I also agree its not always -Infinity